scripts/plot_base_hydrogen_network.py

```diff
@@ -117,7 +168,7 @@ def plot_h2_map_base(network):
         snakemake = mock_snakemake(
             "plot_base_hydrogen_network",
             opts="",
-            clusters="100",
+            clusters="all",
```

Comment:
Maybe good to leave as is
```suggestion
            clusters="100",
```